lib/sqlitex/statement.ex

```diff
@@ -73,6 +73,8 @@ defmodule Sqlitex.Statement do
             column_names: [],
             column_types: []
 
+  @chunk_size 5000
```

Comment:
Where does this value come from?

Reply:
It comes from the `esqlite` project. This is the same default they use. Unfortunately they require you to provide a chunk size in order to provide a timeout.